### PR TITLE
create recipe for ahead

### PR DESCRIPTION
**Pull request by thierrymoudiki**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thierrymoudiki** [comment]
@conda-forge/help-python, ready for review!

File: recipes/ahead/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/Techtonique/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 875c4fa6801865c1e5f18db1ee1edffdea0be9723a77e83e02c46c757ca76b55
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py < 35]
```
Comment (ocefpaf) [inline recipes/ahead/meta.yaml:15]:
```suggestion
  skip: true  # [py<35]
```

Comment (thierrymoudiki) [reply recipes/ahead/meta.yaml:15]: Done

**Comment by thierrymoudiki** [comment]
@ocefpaf Done

File: recipes/ahead/setup.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
+import platform
```
Comment (ocefpaf) [inline recipes/ahead/setup.py:1]: Can you comment on why this file is needed? Is https://github.com/Techtonique/ahead_python/blob/v0.10.0/setup.py different from this one?

Comment (thierrymoudiki) [reply recipes/ahead/setup.py:1]: The only line in diffs is `import platform`? Does it matter much? In any case, this line is not harmful.

Comment (ocefpaf) [reply recipes/ahead/setup.py:1]: It mattes b/c maintaining your own setup.py is easy to get things out of sync and hard to follow the diffs. If you really need that, can you add it as a patch  with a comment on top of it instead? That way we all know the diff and the reason for the patch.

Comment (thierrymoudiki) [reply recipes/ahead/setup.py]: Fails on windows, but windows is not required in the recipe
